src/main/java/com/kobylynskyi/graphql/codegen/model/graphql/GraphQLRequest.java

```diff
@@ -5,14 +5,28 @@
  */
 public class GraphQLRequest {
 
+    private final String operationName;
     private final GraphQLOperationRequest request;
     private final GraphQLResponseProjection responseProjection;
 
     public GraphQLRequest(GraphQLOperationRequest request) {
-        this(request, null);
+        this(null, request, null);
+    }
+
+    public GraphQLRequest(String operationName, GraphQLOperationRequest request) {
+        this(operationName, request, null);
     }
 
     public GraphQLRequest(GraphQLOperationRequest request, GraphQLResponseProjection responseProjection) {
+        this(null, request, responseProjection);
+    }
+
+    public GraphQLRequest(
+        String operationName,
+        GraphQLOperationRequest request,
+        GraphQLResponseProjection responseProjection
+    ) {
```

Comment:
```suggestion
    public GraphQLRequest(String operationName, GraphQLOperationRequest request,
        GraphQLResponseProjection responseProjection) {
```

Reply:
(as per the formating rules located in ./config/checkstyle/graphql-codegen-check-style.xml)